Cannot find module 'CSON'

To reproduce:

1. Create a project (meteor create oneapp)
2. Press ctrl+alt+o

**Atom Version**: 1.5.3
**System**: "Funtoo Linux
**Thrown From**: [mantrajs](https://github.com/tomitrescak/mantrajs-atom-package) package, v0.2.0


### Stack Trace

Failed to activate the mantrajs package

```
At Cannot find module 'CSON'

Error: Cannot find module 'CSON'
    at Module._resolveFilename (module.js:336:15)
    at Function.Module._resolveFilename (/usr/share/atom/resources/app.asar/src/module-cache.js:383:52)
    at Function.Module._load (module.js:286:25)
    at Module.require (module.js:365:17)
    at require (/usr/share/atom/resources/app.asar/src/native-compile-cache.js:50:27)
    at Object.<anonymous> (/home/fermuch/.atom/packages/mantrajs/lib/configHandler.coffee:2:8)
    at Object.<anonymous> (/home/fermuch/.atom/packages/mantrajs/lib/configHandler.coffee:1:1)
    at Module._compile (/usr/share/atom/resources/app.asar/src/native-compile-cache.js:103:30)
    at Object.defineProperty.value [as .coffee] (/usr/share/atom/resources/app.asar/src/compile-cache.js:208:21)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (/usr/share/atom/resources/app.asar/src/native-compile-cache.js:50:27)
    at Object.<anonymous> (/home/fermuch/.atom/packages/mantrajs/lib/mantra-view.coffee:6:10)
    at Object.<anonymous> (/home/fermuch/.atom/packages/mantrajs/lib/mantra-view.coffee:1:1)
    at Module._compile (/usr/share/atom/resources/app.asar/src/native-compile-cache.js:103:30)
    at Object.defineProperty.value [as .coffee] (/usr/share/atom/resources/app.asar/src/compile-cache.js:208:21)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (/usr/share/atom/resources/app.asar/src/native-compile-cache.js:50:27)
    at Object.<anonymous> (/home/fermuch/.atom/packages/mantrajs/lib/mantrajs.coffee:3:14)
    at Object.<anonymous> (/home/fermuch/.atom/packages/mantrajs/lib/mantrajs.coffee:1:1)
    at Module._compile (/usr/share/atom/resources/app.asar/src/native-compile-cache.js:103:30)
    at Object.defineProperty.value [as .coffee] (/usr/share/atom/resources/app.asar/src/compile-cache.js:208:21)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (/usr/share/atom/resources/app.asar/src/native-compile-cache.js:50:27)
    at Package.module.exports.Package.requireMainModule (/usr/share/atom/resources/app.asar/src/package.js:720:27)
    at Package.module.exports.Package.activateNow (/usr/share/atom/resources/app.asar/src/package.js:172:16)
    at /usr/share/atom/resources/app.asar/src/package.js:800:25
    at Function.module.exports.Emitter.simpleDispatch (/usr/share/atom/resources/app.asar/node_modules/event-kit/lib/emitter.js:25:14)
    at Emitter.module.exports.Emitter.emit (/usr/share/atom/resources/app.asar/node_modules/event-kit/lib/emitter.js:125:28)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/usr/share/atom/resources/app.asar/src/command-registry.js:241:20)
    at /usr/share/atom/resources/app.asar/src/command-registry.js:3:61
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/usr/share/atom/resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:536:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/usr/share/atom/resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:359:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeydown (/usr/share/atom/resources/app.asar/src/window-event-handler.js:97:36)
    at HTMLDocument.<anonymous> (/usr/share/atom/resources/app.asar/src/window-event-handler.js:3:61)
```

### Commands

```
     -0:00.8.0 mantrajs:toggle (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "autoHideMenuBar": true,
    "disabledPackages": [
      "jshint",
      "linter-flow"
    ],
    "projectHome": "/home/fermuch/Documentos/Dev/Work",
    "themes": [
      "one-dark-ui",
      "atom-material-syntax"
    ]
  }
}
```

### Installed Packages

```coffee
# User
atom-jasmine, v0.8.1
atom-material-syntax, v0.4.3
cson, v2.0.1
linter, v1.11.3
linter-eslint, v6.0.0
linter-js-yaml, v1.2.5
mantrajs, v0.2.0
minimap, v4.19.0
react, v0.14.1
tabs-to-spaces, v1.0.2

# Dev
No dev packages
```

--- a/lib/configHandler.coffee
+++ b/lib/configHandler.coffee
@@ -1,5 +1,5 @@
 fs = require 'fs'
-cson = require('CSON')
+cson = require('cson')
 
 module.exports =
 class ConfigHandler

